src/aind_metadata_service/routers/perfusions/route.py

```diff
@@ -0,0 +1,121 @@
+"""Module to handle perfusions endpoint responses"""
+
+import json
+import logging
+from typing import Any, Dict, List, Union
+
+from aind_data_schema.core.procedures import Surgery
+from aind_smartsheet_api.client import SmartsheetClient
+from fastapi import APIRouter, Depends, Path, status
+from fastapi.responses import JSONResponse
+from pydantic import ValidationError
+
+from aind_metadata_service.backends.smartsheet.configs import (
+    SmartsheetAppConfigs,
+    get_settings,
+)
+from aind_metadata_service.backends.smartsheet.handler import SessionHandler
+from aind_metadata_service.backends.smartsheet.models import PerfusionsModel
+from aind_metadata_service.backends.smartsheet.session import get_session
+from aind_metadata_service.responses import (
+    InternalServerError,
+    Message,
+    NoDataFound,
+)
+from aind_metadata_service.routers.perfusions.mapper import Mapper
+
+router = APIRouter()
+
+
+class PerfusionsResponse(Message):
+    """Valid Subject response"""
+
+    message: str = "Valid Model"
+    data: Surgery
+
+
+class InvalidPerfusionsResponse(Message):
+    """Invalid Perfusions response"""
+
+    message: str
+    data: Surgery
+
+
+class MultipleItemsFound(Message):
+    """Multiple Items Found response"""
+
+    message: str = "Multiple Items Found"
+    data: List[Union[Surgery, Dict[str, Any]]]
+
+
+@router.get(
+    "/perfusions/{subject_id}",
+    response_model=PerfusionsResponse,
+    responses={
+        status.HTTP_300_MULTIPLE_CHOICES: {"model": MultipleItemsFound},
+        status.HTTP_404_NOT_FOUND: {"model": NoDataFound},
+        status.HTTP_406_NOT_ACCEPTABLE: {"model": InvalidPerfusionsResponse},
+        status.HTTP_500_INTERNAL_SERVER_ERROR: {"model": InternalServerError},
+    },
+)
+async def get_funding(
```

Comment:
I think this is from a copy-paste and needs to be re-named to get_perfusions

Reply:
I renamed it to get_perfusions